apps/backend/.env.example

```
@@ -18,6 +18,10 @@ REDIS_URL="redis://localhost:6379/0"
 # HTTP server port (defaults to 3000 if not set)
 PORT="3000"
 
+# Canonical public origin used in generated calendar URLs
+# Avoids trusting client-controlled forwarded headers
+PUBLIC_APP_ORIGIN="http://localhost:4200"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)